plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/schema/SQLSchemaManager.java

```diff
@@ -172,7 +171,16 @@ public void save() throws IOException {
                 log.debug("Process [" + line + "]");
             }
             try (Statement dbStat = connection.createStatement()) {
-                dbStat.execute(line);
+                try {
+                    dbStat.execute(line);
+                } catch (SQLException e) {
+                    //TODO: find a better way to avoid migration errors
+                    // 11 migration version sometimes crashes in h2
+                    log.error("Error during sql migration", e);
+                    log.info("Trying to apply the migration again");
+                    dbStat.execute(line);
+                    log.error("The second attempt was successful");
```

Comment:
And if not?)

Reply:
then we are sad, cry and go to repair the base using hands 🥲